airbyte_cdk/sources/declarative/partition_routers/substream_partition_router.py

```diff
@@ -381,3 +421,16 @@ def get_stream_state(self) -> Optional[Mapping[str, StreamState]]:
     @property
     def logger(self) -> logging.Logger:
         return logging.getLogger("airbyte.SubstreamPartitionRouter")
+
+
+class SafeResponse(requests.Response):
```

Comment:
nit: can we add a description of what is this object? `Safe` doesn't tell me much in this case.

Reply:
Added doc string